public/data/AbhijithGanesh.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name":"Abhijith Ganesh",
+    "bio":"An Aspiring Tech afficionado and a freshman in computer science",
+    "avatar":"https://github.com/AbhijithGanesh.png",
+    "links":[
+        {
+            "name":"Follow me on GitHub",
+            "url":"https://github.com/AbhijithGanesh",
+            "icon":"github"
+        },
+        {
+            "name":"Conenct with me on LinkedIn",
+            "url":"https://linkedin.com/in/AbhijithGanesh14",
+            "icon":"linkedin"
+        },
+        {
+            "name":"Check my website out",
+            "url":"https://AbhijithGanesh.tech",
+            "icon":"web"
+        }
+    ]
+}
```

Comment:
Was this added before in another PR, or it was added in this PR?

Reply:
It was added in the previous PR and I merged it too. I don't know why it reflected and I didn't want to create another branch again. I rebased my repo before merging but it still shows the same changes.

Reply:
I realize that my changes haven't reflected despite Eddie approving them, it will reflect once this merge is approved.